### PR TITLE
[CBR-48] Fix haddock compilation errors

### PR DESCRIPTION
**Pull request by 365andreas**

### CODE REVIEW

**Comment by gromakovsky** [comment]
It seems to duplicate #2072.

**Comment by 365andreas** [comment]
@gromakovsky Yes it is. Should I update #2072 ?
cc @jcmincke

**Comment by gromakovsky** [comment]
I don't know, to be honest. But whoever will merge this PR, should close #2072 or vice versa.

File: explorer/src/Pos/Explorer/ExtraContext.hs [hunk 1]
```diff
@@ -10,12 +10,12 @@ module Pos.Explorer.ExtraContext
 
     , HasGenesisRedeemAddressInfo (..)
     , GenesisRedeemAddressInfo
-    -- ^ Genesis address info
+    -- * Genesis address info
```
Comment (jcmincke) [inline explorer/src/Pos/Explorer/ExtraContext.hs]: Why a '-- *' here instead of '--'?

Comment (365andreas) [reply explorer/src/Pos/Explorer/ExtraContext.hs]: `-- *` Is acceptable and creates a contents table. I can also replace it with `-- `or put the description above the functions with a `-- |`. In my opinion, the best for the html generated by haddock is the latter and the best for someone reading the source code is the former.
@jcmincke which should I prefer?

**Comment by domenkozar** [comment]
Rocksdb will get further dumped by https://github.com/input-output-hk/cardano-sl/pull/2417

**Comment by 365andreas** [comment]
@domenkozar should I delete changes in files (stack.yaml, pkgs/default.nix) ?

**Comment by domenkozar** [comment]
Best to do that, yes :)

**Pull request merged**